microk8s-resources/actions/kubeflow/sa.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
```

Comment:
We need instructions (at least a README) describing how these manifests where created.

Reply:
pod spec set v2 is coming which will add support for config maps, secrets, resources etc